### PR TITLE
Feature/page objects

### PR DESCRIPTION
**Pull request by kristina-mnoyan**
Changed the structure to PageObject based and added 2 more scenarios

### CODE REVIEW

File: src/main/java/pages/Mailbox.java [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+    @FindBy(xpath = "//tr[@class='zA yO']//span[@class='bog']/span")
+    private List<WebElement> subjects;
+
+    public void clickWriteMessageButton() {
+        writeMessageButton.get(0).click();
+    }
+
+    public String getPlannedMailSendingTime(String subject) {
+        String xPath = String.format("(//*[text()='%s']//ancestor::tr[@class='zA yO']//child::div[@class='by1 kE'])[2]", subject);
+//        return driver.findElement(By.xpath(xPath)).getText();
```
Comment (testforham) [inline src/main/java/pages/Mailbox.java:87]: In general it's not a good practice to push commented lines except the case when it's a Javadoc comment.

File: src/main/java/utils/CommonUtils.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.Locale;
+import java.util.stream.Collectors;
+
+@UtilityClass
+public class CommonUtils {
+    private WebDriver driver = DriverFactory.getDriver();
+
+    public boolean isListEmpty(final List list) {
```
Comment (testforham) [inline src/main/java/utils/CommonUtils.java:19]: seems little bit meaningless method as it does nothing more than checking list emptiness which is Collections method by itself

File: src/test/java/BaseTest.java [hunk 3]
```diff
@@ -1,26 +1,13 @@
...
-        chromeDriver = new ChromeDriver(chromeOptions);
-        chromeDriver.manage().timeouts().implicitlyWait(20, TimeUnit.SECONDS);
-    }
-
+    @SneakyThrows
     @AfterClass
     public void browserTearDown() {
-        chromeDriver.quit();
-        chromeDriver = null;
+//        Thread.sleep(200000);
```
Comment (testforham) [inline src/test/java/BaseTest.java:10]: No need.

**Comment by kristina-mnoyan** [comment]
wrong PR

**Pull request closed without merging**